include/deal.II/lac/vector_space_vector.h

```diff
@@ -154,7 +157,7 @@ namespace LinearAlgebra
     virtual size_type size() const = 0;
 
     /**
-     * Return an index set that describes which elements of this vector are
+     * Returns an index set that describes which elements of this vector are
```

Comment:
We're inconsistent in the way we use this, but I think in most places we use "Return this and that" as opposed to "[This function] Returns this and that."